### PR TITLE
p1: update cmake to include more submission files

### PR DESCRIPTION
**Pull request by thelongmarch-azx**
DO NOT PUSH PROJECT SOLUTIONS PUBLICLY.

And especially do NOT open pull requests with project solutions!

Please be considerate of this free educational resource.

### CODE REVIEW

File: src/include/storage/disk/cow_buffer.h [hunk 1]
```diff
@@ -51,7 +52,7 @@ class CoWBuffer {
   }
 
   /**
-   * @brief Removes a page from the CoW buffer.
+   * @brief Removes a page to the CoW buffer.
```
Comment (AlSchlo) [inline src/include/storage/disk/cow_buffer.h:55]: typo re-introduced

**Review by AlSchlo — APPROVED** [review]

File: src/include/storage/disk/cow_buffer.h [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 #include <cstdint>
 #include <cstring>
+#include <unordered_map>
```
Comment (xx01cyx) [inline src/include/storage/disk/cow_buffer.h:17]: we can remove this as `unorder_map` is not needed in this file

File: test/storage/cow_buffer_test.cpp [hunk 3]
```diff
@@ -52,4 +52,4 @@ TEST(CoWBufferTest, ScheduleWriteReadPageTest) {
   cow.Remove(nullptr);  // Shouldn't crash.
 }
 
-}  // namespace bustub
```
Comment (xx01cyx) [inline test/storage/cow_buffer_test.cpp:55]: should have new line at eof

**Review by yliang412 — APPROVED** [review]
LGTM

**Pull request merged**